cpp/tests/io/json/json_quote_normalization_test.cpp

```diff
@@ -38,17 +36,13 @@ void run_test(std::string const& host_input,
               std::string const& expected_host_output,
               char delimiter = '\n')
 {
-  // RMM memory resource
-  std::shared_ptr<rmm::mr::device_memory_resource> rsc =
-    std::make_shared<rmm::mr::cuda_memory_resource>();
-
   auto stream_view  = cudf::test::get_default_stream();
-  auto device_input = rmm::device_buffer(
-    host_input.c_str(), host_input.size(), stream_view, cudf::get_current_device_resource_ref());
+  auto device_input = rmm::device_buffer(host_input.c_str(), host_input.size(), stream_view);
```

Comment:
Shouldn't this also be using `cudf::get_current_device_resource_ref()`?

Reply:
It does by default. This is consistent with other usages of `rmm::device_buffer` in the libcudf gtests.